### PR TITLE
Update framework and podspec to version 1.0.1

### PR DESCRIPTION
**Pull request by boyvanamstel**

### CODE REVIEW

File: Hanson.podspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |spec|
     spec.name                      = "Hanson"
-    spec.version                   = "1.0"
+    spec.version                   = "1.0.1"
```
Comment (jurre) [inline Hanson.podspec]: I would argue that this is a `1.1.0`, from the SemVer docs:

> MINOR version when you add functionality in a backwards-compatible manner

Comment (boyvanamstel) [inline Hanson.podspec]: Hmm, @joostvandijk and I discussed this, but I think you're right.

Comment (boyvanamstel) [inline Hanson.podspec]: https://github.com/blendle/Hanson/pull/11

Somehow you're not popping up as a reviewer @jurre.

Comment (jurre) [inline Hanson.podspec]: Yeah I'm a member of the organisation but I don't think I have write access to the repo

**Pull request closed without merging**